src/test/kotlin/com/personio/synthetics/e2e/E2EBrowserTest.kt

```diff
@@ -23,6 +23,7 @@ class E2EBrowserTest {
             recoveryMessage("Test recovered")
             env("qa")
             tags("synthetics-api")
+            tags(listOf("one", "two"))
```

Comment:
can we pass it as comma separated and handle the list inside of the function, so won't break the existing behaviour

WDYT?